src/datastore/postgres/schema/v9.ts

```diff
@@ -0,0 +1,10 @@
+import { PoolClient } from "pg";
+
+export async function runSchema(connection: PoolClient) {
+    await connection.query(`
+        ALTER TABLE last_seen ADD COLUMN first BIGINT;
+        ALTER TABLE last_seen RENAME COLUMN ts TO last;
+        UPDATE last_seen SET first = last;
+        ALTER TABLE last_seen ALTER COLUMN first SET NOT NULL;
```

Comment:
That *might* explode for all the existing items that don't have a value set.

Reply:
see the line above it; we backfill `first` from `last`